web-local/src/lib/components/search/Search.svelte

```diff
@@ -9,6 +9,13 @@
   /* Input value being searched */
   export let value;
 
+  function handleKeyDown(event) {
+    if (event.code == "Enter") {
+      event.preventDefault();
+      return false;
+    }
```

Comment:
Wondering if this is the best way to handle this. Probably there is more elegant svelte way of handling this @hamilton ?

Reply:
I think this seems like a pretty sensible solution! We don't want to `preventDefault` in any other case (for instance tabbing away from search)